docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_YOLACT.md

```diff
@@ -0,0 +1,189 @@
+The PyTorch* implementation is publicly available in [this GitHub* repository](https://github.com/dbolya/yolact).
+The YOLACT++ model is not supported, because it uses deformable convolutional layers that cannot be represented in ONNX* format.
+
+## Create a Patch File <a name="patch-file"></a>
+
+Before converting the model, you need to create a patch file for the repository.
+The patch modifies the framework code by adding a special command-line argument to the framework options that enables inference graph dumping:
+
+1. Go to a writable directory and create a `YOLACT_onnx_export.patch` file.
+2. Copy the following diff code to the file:
+```git
+From 76deb67d4f09f29feda1a633358caa18335d9e9f Mon Sep 17 00:00:00 2001
+From: "OpenVINO" <openvino@intel.com>
+Date: Fri, 12 Mar 2021 00:27:35 +0300
+Subject: [PATCH] Add export to ONNX
+
+---
+ eval.py                |  5 ++++-
+ utils/augmentations.py |  7 +++++--
+ yolact.py              | 29 +++++++++++++++++++----------
+ 3 files changed, 28 insertions(+), 13 deletions(-)
+
+diff --git a/eval.py b/eval.py
+index 547bc0a..bde0680 100644
+--- a/eval.py
++++ b/eval.py
+@@ -593,9 +593,12 @@ def badhash(x):
+     return x
+ 
+ def evalimage(net:Yolact, path:str, save_path:str=None):
+-    frame = torch.from_numpy(cv2.imread(path)).cuda().float()
++    frame = torch.from_numpy(cv2.imread(path)).float()
++    if torch.cuda.is_available():
++        frame = frame.cuda()
+     batch = FastBaseTransform()(frame.unsqueeze(0))
+     preds = net(batch)
++    torch.onnx.export(net, batch, "yolact.onnx", opset_version=11)
+ 
+     img_numpy = prep_display(preds, frame, None, None, undo_transform=False)
+     
+diff --git a/utils/augmentations.py b/utils/augmentations.py
+index cc7a73a..2420603 100644
+--- a/utils/augmentations.py
++++ b/utils/augmentations.py
+@@ -623,8 +623,11 @@ class FastBaseTransform(torch.nn.Module):
+     def __init__(self):
+         super().__init__()
+ 
+-        self.mean = torch.Tensor(MEANS).float().cuda()[None, :, None, None]
+-        self.std  = torch.Tensor( STD ).float().cuda()[None, :, None, None]
++        self.mean = torch.Tensor(MEANS).float()[None, :, None, None]
++        self.std  = torch.Tensor( STD ).float()[None, :, None, None]
++        if torch.cuda.is_available():
++            self.mean.cuda()
++            self.std.cuda()
+         self.transform = cfg.backbone.transform
+ 
+     def forward(self, img):
+diff --git a/yolact.py b/yolact.py
+index d83703b..f8c787c 100644
+--- a/yolact.py
++++ b/yolact.py
+@@ -17,19 +17,22 @@ import torch.backends.cudnn as cudnn
+ from utils import timer
+ from utils.functions import MovingAverage, make_net
+ 
+-# This is required for Pytorch 1.0.1 on Windows to initialize Cuda on some driver versions.
+-# See the bug report here: https://github.com/pytorch/pytorch/issues/17108
+-torch.cuda.current_device()
+-
+-# As of March 10, 2019, Pytorch DataParallel still doesn't support JIT Script Modules
```

Comment:
```suggestion
-# As of March 10, 2019, Pytorch DataParallel still does not support JIT Script Modules
```

Reply:
This text is part of the patch file, should not be changed, since it is in the original repository.